src/components/Navigation/ConversationsList/ConversationsList.vue

```diff
@@ -38,12 +38,24 @@ export default {
 	components: {
 		Conversation
 	},
+	props: {
+		searchText: {
+			type: String,
+			default: ''
+		}
+	},
 	computed: {
 		conversationsList() {
 			return this.$store.getters.conversationsList
 		},
+		filteredConversationsList() {
+			if (this.searchText === '') {
+				return this.conversationsList
+			}
+			return this.conversationsList.filter(conversation => conversation.displayName.indexOf(this.searchText) !== -1)
+		},
 		sortedConversationsList() {
-			return this.conversationsList.slice().sort(this.sortConversations)
+			return this.filteredConversationsList.slice().sort(this.sortConversations)
```

Comment:
Why not do all this computation in the `conversationsList` computed prop? Or even in sorted but I mean.. Isn't 1 prop enough here?

Reply:
the reason I did it that way is that its only recalculating where necessary. But i guess most of the time the root data is changing anyway, so yeah let me merge them all in one

Reply:
Done